packages/lit-html/rollup.config.js

```diff
@@ -29,4 +29,12 @@ export default litRollupConfig({
     'lit-html',
     'parts',
   ],
+  reservedProperties: [
+    '_$litType$',
```

Comment:
Could we do a pattern instead? This would also enforce that these special properties are named similarly.